components/brave_welcome_ui/style.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+
+import backgroundUrl2x from './assets/background@2x.jpg'
+
+export const MainBox = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100vh;
+  background: url(${backgroundUrl2x}) no-repeat center center;
```

Comment:
@petemill removed media query for retina and kept the 2x image. It is also optimized. 89.9% 178kb